spec/awscr-s3/object_spec.cr

```diff
@@ -3,28 +3,32 @@ require "../spec_helper"
 module Awscr::S3
   describe Object do
     it "is equal to another object if key size and etag are same" do
-      object = Object.new("test", 123, "etag")
-      Object.new("test", 123, "etag").should eq(object)
+      object = Object.new("test", 123, "etag", "2019-03-02T03:09:04.057Z")
+      Object.new("test", 123, "etag", "2019-03-02T03:09:04.057Z").should eq(object)
     end
 
     it "not equal to another object key size and etag" do
-      object = Object.new("test2", 123, "etag")
-      (Object.new("test", 123, "asd") == object).should eq(false)
+      object = Object.new("test2", 123, "etag", "2019-03-02T03:09:04.057Z")
+      (Object.new("test", 123, "asd", "2019-03-02T03:09:04.057Z") == object).should eq(false)
     end
 
     it "has key" do
-      object = Object.new("test", 123, "etag")
+      object = Object.new("test", 123, "etag", "2019-03-02T03:09:04.057Z")
       object.key.should eq("test")
     end
 
     it "has size" do
-      object = Object.new("test", 123, "etag")
+      object = Object.new("test", 123, "etag", "2019-03-02T03:09:04.057Z")
       object.size.should eq(123)
     end
 
     it "has etag" do
-      object = Object.new("test", 123, "etag")
+      object = Object.new("test", 123, "etag", "2019-03-02T03:09:04.057Z")
       object.etag.should eq("etag")
     end
+    it "has etag" do
```

Comment:
Needs a space, above `it` (or just run `crystal tool format`) and test description is repeated :).